homeassistant/components/webostv/media_player.py

```diff
@@ -4,7 +4,8 @@
 import asyncio
 from collections.abc import Awaitable, Callable, Coroutine
 from contextlib import suppress
-from datetime import timedelta
+import datetime as datetime2
+from datetime import datetime, timedelta
```

Comment:
```suggestion
import datetime as datetime2
from datetime import datetime, timedelta
```

```suggestion
from datetime import datetime, timedelta
```

its unnecessary to have both import statements. Later on when you need `strptime()`, call it with `datetime.strptime()`